integration/debug_test.go

```diff
@@ -111,6 +129,89 @@ func TestDebug(t *testing.T) {
 	}
 }
 
+func TestDockerDebug(t *testing.T) {
+	MarkIntegrationTest(t, CanRunWithoutGcp)
+
+	t.Run("debug docker deployment", func(t *testing.T) {
+		skaffold.Build("-p", "docker").InDir("testdata/debug").RunOrFail(t)
+
+		skaffold.Debug("-p", "docker").InDir("testdata/debug").RunBackground(t)
+		defer skaffold.Delete("-p", "docker").InDir("testdata/debug").RunBackground(t)
+
+		// use docker client to verify container has been created properly
+		// check this container and verify entrypoint has been rewritten
+		kubeConfig, err := kubectx.CurrentConfig()
+		if err != nil {
+			t.Log("unable to get current cluster context: %w", err)
+			t.Logf("test might not be running against the right docker daemon")
+		}
+		kubeContext := kubeConfig.CurrentContext
+		// log.Entry(context.TODO()).Infof("Using kubectl context: %s", kubeContext)
+		// client, err := docker.DefaultAPIClient(kubeContext)
+		client, err := docker.NewAPIClient(context.Background(), debugConfig{kubeContext: kubeContext})
```

Comment:
It would be worth extracting this to a helper function: I thought this was somehow relevant to the test and was confused.

Reply:
done